diff --git a/scripts/jqf-driver.sh b/scripts/jqf-driver.sh
index 568c5109..b97043cd 100755
--- a/scripts/jqf-driver.sh
+++ b/scripts/jqf-driver.sh
@@ -19,7 +19,7 @@ INST_JAR="${INST_DIR}/$project-instrument-$version.jar"
 
 # Compute classpaths (the /classes are only for development; 
 #   if empty the JARs will have whatever is needed)
-INST_CLASSPATH="${INST_DIR}/classes:${INST_JAR}:${INST_DIR}/dependency/asm-9.8.jar"
+INST_CLASSPATH="${INST_DIR}/classes:${INST_JAR}:${INST_DIR}/dependency/asm-9.10.1.jar"
 FUZZ_CLASSPATH="${FUZZ_DIR}/classes:${FUZZ_JAR}"
 
 # If user-defined classpath is not set, default to '.'
